HipoProject/urls.py

```diff
@@ -21,13 +21,14 @@
 # from users.views import SignUp
 # from django.contrib.auth.decorators import login_required
 from rest_framework import routers
-from users.views import UserRegistrationView, UserAuthenticationView, MyProfileDetailView
+from users.views import UserRegistrationView, UserAuthenticationView, MyProfileDetailView, ChangePasswordView
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/profiles/create/', UserRegistrationView.as_view(), name='create_user'),
     path('api/profiles/authenticate/', UserAuthenticationView.as_view()),
     path('api/profiles/me/', MyProfileDetailView.as_view()),
+    path('api/profiles/change_password/', ChangePasswordView.as_view()),
```

Comment:
We don't use underscore `_` in uri's. Can you please use `-` ? It should be `api/profiles/me/change-password/`.